tests/manage/pv_services/pvc_clone/test_pvc_to_pvc_clone.py

```diff
@@ -136,3 +139,75 @@ def test_pvc_to_pvc_clone(self, interface_type, teardown_factory):
         # Wait for IO to finish on the new pod
         clone_pod_obj.get_fio_results()
         logger.info(f"IO completed on pod {clone_pod_obj.name}")
+
+    @pytest.mark.polarion_id("OCS-5162")
+    @pytest.mark.parametrize(
+        argnames=["interface_type", "pod_dict_path"],
+        argvalues=[
+            pytest.param(constants.CEPHFILESYSTEM, constants.CSI_CEPHFS_ROX_POD_YAML),
+        ],
+    )
+    def test_pvc_to_pvc_rox_clone(
+        self, snapshot_factory, snapshot_restore_factory, teardown_factory
+    ):
+        """
+        Create a rox clone from an existing pvc,
+        verify data is preserved in the cloning.
+        """
+        logger.info(f"Running IO on pod {self.pod_obj.name}")
+        file_name = f"{self.pod_obj.name}.txt"
+        self.pod_obj.exec_cmd_on_pod(
+            command=f"dd if=/dev/zero of=/mnt/{file_name} bs=1M count=1"
```

Comment:
Are we planing any enhancement to add md5sum validation?

Reply:
currently not as the pod is not supporting those actions, will try to work on it and see if I get an alternative for that. will create a new pr then